### ISSUE TITLE
Mismatch between readme and configuration for home-row mods

### ISSUE THREAD

**Issue by PhML** [OneDeadKey/arsenik#53]
Readme says `FDS` and `JKL` are mapped to  `Ctrl` , `Alt`, `Super` but in fact it is , `Alt` , `Ctrl`,  `Super`.

**Comment by Erwyn** [comment]
I confirm that I do encounter the same behavior.

**Comment by Nuclear-Squid** [comment]
Yep, we changed the order not too long after the start of the project and forgot to update the readme. Sorry for the confusion.
